application-deployment/jenkins/jenkins-java.nomad

```diff
@@ -33,26 +33,25 @@ job "jenkins" {
         args        = ["--httpPort=8080"]
       }
       artifact {
-        source = "http://ftp-chi.osuosl.org/pub/jenkins/war-stable/2.46.2/jenkins.war"
```

Comment:
This no longer exists. Additionally, I thought using Jenkins' official mirror would be more appropriate.